code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -395,17 +403,25 @@
 			if("medical")
 				return istype(H.glasses, /obj/item/clothing/glasses/hud/health) || istype(CIH,/obj/item/organ/internal/cyberimp/eyes/hud/medical)
 			else
-				return 0
+				return FALSE
 	else if(isrobot(M) || isAI(M)) //Stand-in/Stopgap to prevent pAIs from freely altering records, pending a more advanced Records system
 		switch(hudtype)
 			if("security")
-				return 1
+				return TRUE
 			if("medical")
-				return 1
+				return TRUE
 			else
-				return 0
+				return FALSE
+	else if(istype(M, /mob/dead/observer))
+		var/mob/dead/observer/O = M
+		if(O.data_hud_seen == DATA_HUD_SECURITY_ADVANCED || O.data_hud_seen == DATA_HUD_DIAGNOSTIC + DATA_HUD_SECURITY_ADVANCED + DATA_HUD_MEDICAL_ADVANCED)
```

Comment:
Wouldn't it make more sense to do `O.data_hud_seen & DATA_HUD_SECURITY_ADVANCED` here?

Reply:
wait no, it looks like we have the ghost "omni-hud" work as a special identifier derived from adding them together.
why do we do it this way this is so cursed
I guess this will have to do for now